fcrepo-http-api/src/main/java/org/fcrepo/http/api/responses/StreamingBaseHtmlProvider.java

```diff
@@ -187,35 +186,26 @@ void init() throws IOException {
     }
 
     @Override
-    public void writeTo(final RdfStream rdfStream, final Class<?> type,
+    public void writeTo(final RdfNamespacedStream nsStream, final Class<?> type,
                         final Type genericType, final Annotation[] annotations,
                         final MediaType mediaType,
                         final MultivaluedMap<String, Object> httpHeaders,
                         final OutputStream entityStream) throws IOException {
 
-        try {
-            final RdfStream nsRdfStream = new NamespaceRdfContext(rdfStream.session());
-
-            rdfStream.namespaces(nsRdfStream.namespaces());
-
-            final Node subject = VIEW_HELPERS.getContentNode(rdfStream.topic());
-
-            final Model model = rdfStream.asModel();
+        final Node subject = VIEW_HELPERS.getContentNode(nsStream.stream.topic());
```

Comment:
RdfNamespacedStream is simply a Tuple with two members. I would really prefer that it is no more complex than that.

This is a consequence of removing the namespaces() method from RdfStream (previously, it had been carrying around a jcr.Session). This structure (RdfNamespacedStream) sets the stage for injecting a NamespaceFactory, which is not included here because it would require changing the Spring configuration in the webapp and therefore a major version bump.